conda_build/post.py

```diff
@@ -320,7 +320,7 @@ def mk_relative(m, f):
     if not is_obj(path):
         return
 
-    if sys.platform.startswith('linux'):
+    if sys.platform.startswith('linux') or sys.platform.startswith('openbsd'):
```

Comment:
It _might_ work the same on other BSDs, but I don't have a Net/Free BSD to test. Also, sys.platform form e.g. OpenBSD 5.x is `"openbsd5"`, so matching `endswith("bsd")` wouldn't even work here.